dev-docs/source/Testing/EngineeringDiffraction/EngineeringDiffractionTestGuide.rst

```diff
@@ -143,103 +143,110 @@ Test 4
 
 This test covers the loading and plotting focused data in the fitting tab.
 
-0. Create a new calibration, untick ``Crop calibration`` enter calibration sample # ``305738`` and click Calibrate.
-   On the ``Focus`` tab, set ``Sample Run #`` to ``305793-305795`` and ``Vanadium #`` to ``307521`` and click ``Focus``. These sample runs have different stress and strain log values.
+.. note:: Sometimes it will be tricky to load ENGINX files from the archive and the red ``*`` next to the ``Browse`` button won't disappear and will raise an error saying ``Check run numbers/path is valid.`` or ``Mantid is searching for data files. Please wait``. Try re-entering the Calibration Sample number and wait till the red ``*`` is cleared before clicking ``Calibrate`` button. If the log level is set to Information, found path = 1 will be visible when the runs are found from the archive.
 
-1. In the fitting tab, load multiple of these newly focused TOF .nxs files in the `Load Focused Data` section. The path to the focused files should be auto populated
+0. Ensure you can access the ISIS data archive. In the ``Calibration`` tab, select ``Create New Calibration`` and enter ``Calibration sample`` # ``305738``. Before proceeding, make sure the red ``*`` next to the ``Browse`` button is disappeared when clicked somewhere outside that text box.
```

Comment:
Thanks for drawing attention to the need to wait for the run to be found. But would it be possible to split this into two items: one for the calibration, one for the focussing? Also I can't remember now why it's necessary to re-do to calibration rather than loading an existing one?